### PR TITLE
Completed CSS-Positioning

### PR DESCRIPTION
**Pull request by Henry-Nichols**
image
image
image

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  top: 5px;
+  left: 230px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:23]: This is a big movement. The emoji is against the right side of the `.meme` element, you may want to start from the `right`

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  top: 160px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:37]: This is a big movement. The joke is against the bottom of the `.meme` element, you may want to start from the `bottom`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! Please see my notes

**Comment by Henry-Nichols** [comment]
Oh I should've caught on to that. I will correct that right now thank you for the feedback Cody!

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**